add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1374,13 +1373,15 @@ this.MozLoopService = {
       }
 
       let window = gWM.getMostRecentWindow("navigator:browser");
+      if (window.LoopUI.getCurrentRoomToken() && window.LoopUI.getCurrentRoomToken() === room.roomToken) {
```

Comment:
This might want a comment that this is to only show the infobar if this event is for the current room.